helm/hpcc/values.yaml

```diff
@@ -84,8 +84,18 @@ global:
       type: LoadBalancer
       
       
-
-  # For pod placement instruction and examples please reference docs/placements.md
+# For pod placement instruction and examples please reference docs/placements.md
+# The following is for tolerations of Spot Node Pool on Azure. Other cloud providers
+# may have different taints for Spod Node Pool. The tolerations are harmless when
+# there is no taint on the node pool.
+placements:
```

Comment:
SHould this section be commented out, to serve as an example, rather than defaulting all deployments to have it?

Reply:
OK. I comment it out